IonDotnet/IonDotnet.csproj

```diff
@@ -5,12 +5,15 @@
     <RootNamespace>IonDotnet</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <LangVersion>7.3</LangVersion>
-    <PackageId>IonDotnet</PackageId>
-    <Version>0.0.3-alpha</Version>
-    <Authors>hdhoang</Authors>
+    <PackageId>AmazonIonDotnet</PackageId>
```

Comment:
I'd suggest we follow the [dot namespacing convention](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#choose-a-unique-package-identifier-and-setting-the-version-number) for NuGet; see also ["Amazon" packages on NuGet](https://www.nuget.org/packages?q=Amazon).

So:  `Amazon.IonDotnet` (and `Amazon.IonHashDotnet`)?

Reply:
`Amazon.IonDotnet` is fine. I suppose this means we should change the namespace of every source file to `Amazon.IonDotnet` too to follow the convention of keeping it in line with the PackageId.